### PR TITLE
misc fixes

### PR DESCRIPTION
**Pull request by turadg**
## Description

Some things I ran into during recent PRs:

- refactor: StorageNodeShape export
- refactor: simplify makeCollectFeesInvitation
- fix(types): makeStoreUtils return types
- fix(zcf): catch initSeatMgrAndMintFactory rejections
- chore(types): better EV type

### CODE REVIEW

File: packages/inter-protocol/src/collectFees.js [hunk 1]
```diff
@@ -2,18 +2,14 @@ import { atomicTransfer } from '@agoric/zoe/src/contractSupport/index.js';
 
 /**
  * Provide shared support for providing access to fees from a service contract.
- *
+
```
Comment (michaelfig) [inline packages/inter-protocol/src/collectFees.js]: I think this was an accident.

File: packages/vat-data/src/vat-data-bindings.js [hunk 2]
```diff
@@ -134,16 +134,16 @@ harden(partialAssign);
...
- * @param {K} key
- * @param {(key: K) => V} makeValue
- * @returns {V}
  */
-export const provide = provideLazy;
+
+export const provide =
+  // XXX cast because provideLazy is `any` due to broken type import
+  /** @type {<K, V>(baggage: import('./types.js').Baggage, key: K, makeValue: (key: K) => V) => V} */ (
+    provideLazy
```
Comment (michaelfig) [inline packages/vat-data/src/vat-data-bindings.js:142]: Do you have any ideas why the import isn't working?  It would be nice to document that somewhere.

**Review by michaelfig — APPROVED** [review]

File: packages/vat-data/src/vat-data-bindings.js (hunk 2 above)
Comment (turadg) [reply packages/vat-data/src/vat-data-bindings.js:142]: wish I did but nope

**Comment by turadg** [comment]
@mergifyio refresh

**Pull request merged**